mcs/tools/mdoc/Mono.Documentation/monodocer.cs

```diff
@@ -2897,6 +2947,15 @@ static class DocUtils {
 			element.SetAttribute ("apistyle", styleString);
 		}
 	}
+	public static void RemoveApiStyle (this XmlElement element, ApiStyle style) {
+		element.RemoveAttribute (style.ToString ().ToLower ());
```

Comment:
*always* use `.ToLowerInvariant()`, unless you want your code to break in a Turkish locale.